modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -5602,7 +5602,7 @@ void GDScriptAnalyzer::resolve_pending_lambda_bodies() {
 }
 
 bool GDScriptAnalyzer::class_exists(const StringName &p_class) const {
-	return ClassDB::class_exists(p_class) && ClassDB::is_class_exposed(p_class);
+	return (ClassDB::class_exists(p_class) && ClassDB::is_class_exposed(p_class)) || p_class=="Editor";
```

Comment:
This is incorrect. The issue is about displaying **custom** classes in the editor dialog, and you are trying to fix it in the analyzer method that checks **native** classes. Also, this is unrelated to the PR title.